lua/rocks-config/init.lua

```diff
@@ -27,10 +27,8 @@ function rocks_config.setup(user_configuration)
 
             local ok, err = pcall(require, search)
 
-            if not ok and not err:match("module%s+." .. search:gsub("%p", "%%%1") .. ".%s+not%s+found") then
+            if not ok and type(err) ~= "boolean" and not err:match("module%s+." .. search:gsub("%p", "%%%1") .. ".%s+not%s+found") then
```

Comment:
```suggestion
            if not ok and type(err) == "string" and not err:match("module%s+." .. search:gsub("%p", "%%%1") .. ".%s+not%s+found") then
```

Reply:
Looks way better to me, thanks for the suggestion!